react/tic-tac-toe/src/redux/game/reducer.js

```diff
@@ -0,0 +1,35 @@
+import { actions } from './actions';
+
+const initialState = {
+  history: [Array(9).fill(null)],
+  stepNumber: 0,
+  xIsNext: true,
+  winner: null
+};
+
+function reducer(state = initialState, action) {
+  switch (action.type) {
+    case actions.ADD_MOVE:
+      return {
```

Comment:
Mmm for do this you should use `return state.merge({`. This way you merge the new param in the old state